extensions/adobe/experience/adcloud/search/account-all.schema.json

```diff
@@ -0,0 +1,75 @@
+{
+    "meta:license": [
+        "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+        "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+        "you may not use this file except in compliance with the License. You may obtain a copy",
+        "of the License at https://creativecommons.org/licenses/by/4.0/"
+    ],
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "$id": "https://ns.adobe.com/experience/adcloud/search/account-all",
+    "title": "Adobe AdCloud Search Account Entity Full Extension",
+    "type": "object",
+    "meta:extensible": true,
+    "meta:abstract": true,
+    "meta:intendedToExtend": [
+        "https://ns.adobe.com/experience/adcloud/search/profile-adcloud"
```

Comment:
Would suggest change this to https://ns.adobe.com/xdm/context/profile. Will be removing the identityMap from this core class in another PR.

Reply:
Modified this across all objects to use base profile class pre-defined in XDM registry.